### PR TITLE
Feature/zenko 1429 cifs docs

### PR DESCRIPTION
**Pull request by ssalaues**
**What does this PR do, and why do we need it?**
Adds doc on how to manually setup CIFS on the Zenko/MetalK8s stack

**Which issue does this PR fix?**

ZENKO-1422

**Special notes for your reviewers**:

### CODE REVIEW

File: docs/Installation/topics/adding_CIFS_storage.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+4. Use information from the previous steps to configure instance-specific Cosmos values. Export the following variables with appropriate values entered:
+--------------------------------------------------------------------------------------------------------------------------------------------------------
+
+.. code:: bash
+
+   # Values from Orbit
+   export ACCESS_KEY=<your-cosmos-user-access-key>
+   export SECRET_KEY=<your-cosmos-user-secret-key>
+   export SMB_BUCKET=<your-cosmos-bucket-name>
+   export SMB_LOCATION=<your-nfs-mount-location-name>
```
Comment (giacope) [inline docs/Installation/topics/adding_CIFS_storage.rst]: It's right but it's confusing

File: docs/Installation/topics/adding_CIFS_storage.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+   # Values of your NFS mount point
```
Comment (giacope) [inline docs/Installation/topics/adding_CIFS_storage.rst]: SMB share?

File: docs/Installation/topics/adding_CIFS_storage.rst [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+   # Cloudserver endpoint (assuming it's running on the same namespace)
+   export CLOUDSERVER_ENDPOINT="http://$(kubectl get svc -l app=cloudserver -o jsonpath='{.items[*].metadata.name}')"
+
+5. Create a secret to store the CIFS/SMB account name and password
+------------------------------------------------------------------
+
+.. code:: bash
+
+   kubectl create secret generic smbcreds --from-literal username=<USERNAME> --from-literal password="<PASSWORD>" --type="microsoft.com/smb"
```
Comment (giacope) [inline docs/Installation/topics/adding_CIFS_storage.rst]: `<USERNAME>` doesn't need to be quoted to be consistent with `"<PASSWORD>"`?

Comment (ssalaues) [reply docs/Installation/topics/adding_CIFS_storage.rst]: I'll make both quoted. I had issues when testing with special characters that weren't quoted.

**Review by giacope — APPROVED** [review]

**Comment by wabernatScality** [comment]
@bert-e approve

**Review by wabernat — APPROVED** [review]

**Comment by ssalaues** [comment]
@bert-e approve

**Pull request merged**